src/Jobs/Concerns/HasEmitterConcern.php

```diff
@@ -13,10 +34,11 @@ trait HasEmitterConcern
     public function log($message, $context)
     {
         if (config('event-emitter.logging', false)) {
+            $source = $this->getSource() ?? 'undefined';
             $context = array_merge($context, [
                 'authUser' => data_get($this->getAuthUser(), 'id'),
             ]);
-            logger()->info("[Event Emitter] {$message}", $context);
+            logger()->info("[Event Emitter] Source $source {$message}", $context);
```

Comment:
@carroaung should we add the source as one items of log context, like the authUser? 🤔

Reply:
https://github.com/carro-public/event-emitter/pull/4/commits/5cd96e18d3ef4834798e9f758100621a7a935cab